apps/teams-test-app/e2e-test-data/appNotification.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name":"AppNotification",
+  "version":">=2.14.0",  
+  "platforms": "Web",
+  "checkIsSupported": {
+    "domElementName": "checkappNotificationCapability"
+  },
+
+  "testCases":[
+   {
+    "title": "DisplayAppNotification API call- Success",
+    "type": "callResponse",
+    "boxSelector": "#box_checkAppNotificationCapability",
+    "inputValue": {
+      "title": "sampleAppNotificationTitle",
+      "content": "sampleAppNotficationContent",
+      "displayDurationInSeconds": 30,
+      "notificationActionUrl": "https://www.someAppNotificationUrl.com"
```

Comment:
does this Url work when you try locally? I think it should not pass the domain validation

Reply:
No it didn't, I've updated it to a URL which would pass the domain validation.